sys/include/net/ng_pkt.h

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (C) 2014 Martin Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    ng_pkt Packet
+ * @brief       Network packet abstraction types
+ * @ingroup     net
+ * @{
+ *
+ * @file
+ * @brief   General definitions for network packets
+ *
+ * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+#ifndef __NG_PKT_H_
+#define __NG_PKT_H_
+
+#include <inttypes.h>
+#include <stdlib.h>
+
+#include "net/ng_nettype.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Type to represent parts (snips) of a network packet
```

Comment:
Maybe introduce "snip" here a bit more explicitly and use it throughout the remainder of the documentation.